### PR TITLE
Transfer request builder

### PR DESCRIPTION
**Pull request by nuxzero**
Implement the new request builder for the transfer request.

### CODE REVIEW

File: src/main/java/co/omise/models/Transfer.java [hunk 1]
```diff
@@ -133,42 +142,128 @@ public void setMetadata(Map<String, Object> metadata) {
...
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "transfers", transferId);
+        }
+
+        @Override
+        protected String method() {
+            return PATCH;
+        }
```
Comment (Namiii) [inline src/main/java/co/omise/models/Transfer.java]: Shouldn't payload() also be overriden here?

File: src/main/java/co/omise/models/Transfer.java [hunk 2]
```diff
@@ -133,42 +142,128 @@ public void setMetadata(Map<String, Object> metadata) {
...
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "transfers");
+        }
+
+        @Override
+        protected String method() {
+            return POST;
+        }
```
Comment (Namiii) [inline src/main/java/co/omise/models/Transfer.java]: payload()?

File: src/main/java/co/omise/models/Transfer.java (hunk 1 above)
Comment (nuxzero) [reply src/main/java/co/omise/models/Transfer.java]: @Namiii Look like it worked without call `payload()`.

Comment (nuxzero) [reply src/main/java/co/omise/models/Transfer.java]: @Namiii Tested, it's not work. I'll add `payload()`. 😅

Comment (Namiii) [reply src/main/java/co/omise/models/Transfer.java]: 😆

**Comment by nuxzero** [comment]
@Namiii Fixed, please review again 😂

**Pull request merged**